XamlControlsGallery/.editorconfig

```diff
@@ -0,0 +1,109 @@
+﻿# EditorConfig  https://EditorConfig.org
+# Style as Code Minimal EditorConfig v.2020.02.04.0 https://styleascode.net
+
+# top-most EditorConfig file
+root = true
+
+# Uses spaces for indentation.
+[*]
+indent_style = space
+
+# Code files
+[*.{cs,csx}]
```

Comment:
What about xaml files? 😅

Reply:
XAML files can't be analyzed via Roslyn Analyzers, so they don't have any rules that can appear in EditorConfig files.

There is a XAML Analysis package, but it is in beta, can't be configured, and has some issues from when I tried it out.

Reply:
What about indent size and such?

Reply:
Those can be handled, but I was really just trying to tackle C# rules for now in the theme of the minimal editorconfig. Example from microsoft-ui-xaml:

```
[**.{xaml,xml}]
indent_style = space
indent_size = 4
charset = utf-8-bom
insert_final_newline = true
trim_trailing_whitespace = true
end_of_line = crlf
```

Reply:
Since you already have a small snippet for .cs files, adding a few lines for XAML wouldn't hurt ^^

Reply:
Bumped the config version with a few changes to include XAML & XML, character encoding, whitespace trimming, and removal of a rule requires another to be present (it isn't).